HospitalToday/Repository/MedicineRepository.cs

```diff
@@ -0,0 +1,63 @@
+﻿using HospitalToday.Domain;
+using HospitalToday.Domain.Medicine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalToday.Repository
+{
+    class MedicineRepository : IRepository<Medicine>
+    {
+        private static IRepository<Medicine> medRep;
+        private List<Medicine> medicines;
+
+        private MedicineRepository()
+        {
+            medicines = new List<Medicine>();
+        }
+
+        public static IRepository<Medicine> GetRepository()
+        {
+            if (medRep == null)
+                medRep = new MedicineRepository();
+
+            return medRep;
+        }
+
+        public void Create(Medicine item)
+        {
+            medicines.Add(item);
+        }
+
+        public void Delete(int id)
+        {
+            var t = medicines.FirstOrDefault(m => m.Id == id);
```

Comment:
It is ok, but it will have better look if you will use Any().
If you will decide to leave this solution, please rename `t` to give it more informative name.

Reply:
Fixed. I added check when i use Any().